src/Operations.js

```diff
@@ -767,9 +774,11 @@ export function zipWithFactory(keyIter, zipper, iters) {
         steps = iterators.map(i => i.next());
         isDone = steps.some(s => s.done);
```

Comment:
I *think* that this is the critical point. a zipAll is done when *all* steps are done instead of *some* steps are done.